test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/RuntimeTagHelperDescriptorFactoryTest.cs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNet.Razor.TagHelpers;
+using Microsoft.AspNet.Razor.Test.Internal;
+using Xunit;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class RuntimeTagHelperDescriptorFactoryTest : TagHelperDescriptorFactoryTest
+    {
+        public override ITypeInfo GetTypeInfo(Type tagHelperType) =>
+            new RuntimeTypeInfo(tagHelperType.GetTypeInfo());
+
+        [Fact]
+        public void CreateDescriptors_BuildsDescriptorsFromSimpleTypes()
```

Comment:
I guess we could - the way the tests are structured, we try to read a resource and compile it to produce the type resolver. It's kinda messier to do when it's some random object (we have nothing to compile?). I can try and make it work if we really care to.